cibyl/cli/main.py

```diff
@@ -122,10 +122,9 @@ def main() -> None:
         orchestrator.validate_environments()
         features = orchestrator.load_features()
         orchestrator.query_and_publish(
-            output_path=arguments["output_file_path"],
-            output_style=arguments["output_style"],
-            features=features
-        )
+                output_path=arguments["output_file_path"],
+                output_style=arguments["output_style"],
+                features=features)
```

Comment:
Unnecessary change, but well, it does not really matter.

Reply:
While trying things out I removed the arguments and added them back, while editing I must have moved the parenthesis without realising. I'm surprise it marks the three lines as changed though.